starters/starters.yaml

```diff
@@ -421,4 +421,13 @@
   preview: https://akarui.netlify.app
   author: arturoalviar
   tags: gridsome, starter, sanity, blog, portfolio, tailwind, tailwindcss, dark theme, responsive
-  platforms: sanity
\ No newline at end of file
+  platforms: sanity
+
+- title: Gridsome Algrid
+  description: Algrid is a simple starter for creating blogs using Gridsome.
```

Comment:
Could you perhaps update this description to include the blog features, like using Algolia and implementing infinite scroll?

Reply:
@thetre97 I've done to update the description! Could you check please?